tests/rados/pool_tests.py

```diff
@@ -134,12 +134,24 @@ def run(ceph_cluster, **kw):
         compression_config = config["Compression_tests"]["compression_config"]
         pool_1 = pool_config["pool-1"]
         pool_2 = pool_config["pool-2"]
-        if not rados_obj.create_pool(pool_name=pool_1, **pool_config):
-            log.error("could not create pool-1")
-            return 1
-        if not rados_obj.create_pool(pool_name=pool_2, **pool_config):
-            log.error("could not create pool-2")
-            return 1
+
+        if config["Compression_tests"]["pool_type"] == "replicated":
+            if not rados_obj.create_pool(pool_name=pool_1, **pool_config):
```

Comment:
Would it make sense to have a generic `create_pool` method and have it create `ec-pool` based on the argument provided to it else a regular osd pool.

Reply:
Hi Satya, the ` create_pool ` method is already generic, and creates the pool, replicated or erasure, based on the args passed. What ` create_erasure_pool ` method does is, it creates a new EC profile for the pool to be created, verifies that the profile is created successfully, and then calls the same ` create_pool `  method, supplying it with the profile created.

Reply:
Thanks for details. It makes sense to have different methods.